### PR TITLE
GRO-862: Signed out user flow error for preference center

### PR DESCRIPTION
**Pull request by The-Beez-Kneez**
[GRO-862]
Added: Resolving authentication token not being passed properly for Preference Center updated. Being updated in Braze, but throwing an error that is citing a lack of user.

Need to see why toast is throwing us this error, even though Braze is successfully updating.

[GRO-862]: https://artsyproduct.atlassian.net/browse/GRO-862?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by The-Beez-Kneez** [comment]
@jonallured making sure we have an active WIP open to aid in discussions

File: src/v2/Apps/_Preferences2/PreferencesApp.tsx [hunk 1]
```diff
@@ -67,8 +70,10 @@ export const PreferencesApp: FC<PreferencesAppProps> = ({ viewer }) => {
               }
             )
 
+            const input = { authenticationToken, subscriptionGroups }
```
Comment (damassi) [inline src/v2/Apps/_Preferences2/PreferencesApp.tsx:73]: #superminor - might as well just avoid the extra variable here

**Review by dzucconi — APPROVED** [review]
Some specs to fix but 👍

**Review by damassi — DISMISSED** [review]

**Comment by jonallured** [comment]
Ok got it green going to merge!

**Pull request merged**